### PR TITLE
TELCODOCS-321 Release Notes

### PR DESCRIPTION
**Pull request by kdrake20**
Fixes: https://issues.redhat.com/browse/TELCODOCS-321

For: Version 4.10 Release Notes

Doc Preview: https://deploy-preview-42898--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes

Signed-off-by: Katie Tothill ktothill@redhat.com

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -84,6 +84,13 @@ The {product-title} 4.10 installer uses new default component types for installa
 * AWS EC2 M6i instances for both control plane and compute nodes, where available
 * AWS EBS gp3 storage
 
+[id="ocp-4-10-enhancements-to-API-in-installconfig"]
+==== Enhancements to API in install.config
+
+When a user installs OCP on a Baremetal IPI platform, they have nowhere to configure custom network interfaces, such as static IPs or a vLAN to communicate with the Ironic server.
```
Comment (yporagpa) [inline release_notes/ocp-4-10-release-notes.adoc:90]: I feel like the wording on this line is a little misleading.
maybe add "formerly..." or something to clarify this was the case in prior versions.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -84,6 +84,13 @@ The {product-title} 4.10 installer uses new default component types for installa
...
+
+With this feature, for Day 1 install on bare metal only, using a new API change in install-config, the user can customise the network configuration (network-config) per host during the install and provisioning process, including setting static IPs per host and other advanced network configurations.
```
Comment (yporagpa) [inline release_notes/ocp-4-10-release-notes.adoc:92]: Im not sure - but "...network configuration (network-config)..." seemed a little excessive.
are you referring to the new 'networkConfig' field in install-config.yaml? if so you should correct it to the exact capitalization.
if not , are you referring to a different technical element?

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (ktothill) [reply release_notes/ocp-4-10-release-notes.adoc]: Updated as suggested.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 2 above)
Comment (ktothill) [reply release_notes/ocp-4-10-release-notes.adoc]: Updated as suggested.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -84,6 +84,13 @@ The {product-title} 4.10 installer uses new default component types for installa
 * AWS EC2 M6i instances for both control plane and compute nodes, where available
 * AWS EBS gp3 storage
 
+[id="ocp-4-10-enhancements-to-API-in-installconfig"]
+==== Enhancements to API in install.config
+
+Previously, when a user installed OCP on a Baremetal IPI platform, they had nowhere to configure custom network interfaces, such as static IPs or vLANs to communicate with the Ironic server.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-10-release-notes.adoc:90]: We don't use IPI in the docs, can you spell out "installer-provisioned infrastructure?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -84,6 +84,13 @@ The {product-title} 4.10 installer uses new default component types for installa
...
+
+When configuring a Day 1 install on bare metal only, users can now utilize the API in install-config to customise the network configuration (`networkConfig`). This configuration is set during the install and provisioning process and includes advanced options, such as setting static IPs per host.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-10-release-notes.adoc:92]: * install -> installation (two places)
* utilize -> use
* customise -> customize

File: release_notes/ocp-4-10-release-notes.adoc [hunk 5]
```diff
@@ -84,6 +84,13 @@ The {product-title} 4.10 installer uses new default component types for installa
 * AWS EC2 M6i instances for both control plane and compute nodes, where available
 * AWS EBS gp3 storage
 
+[id="ocp-4-10-enhancements-to-API-in-installconfig"]
+==== Enhancements to API in install.config
```
Comment (bergerhoffer) [inline release_notes/ocp-4-10-release-notes.adoc:88]: * in -> in the
* install.config -> install-config? or `install-config.yaml` file?

Comment (ktothill) [reply release_notes/ocp-4-10-release-notes.adoc]: in the `install-config.yaml` file. Updated as suggested.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 3 above)
Comment (ktothill) [reply release_notes/ocp-4-10-release-notes.adoc]: Updated as suggested.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 4 above)
Comment (ktothill) [reply release_notes/ocp-4-10-release-notes.adoc]: Updated as suggested.

**Review by bergerhoffer — COMMENTED** [review]
LGTM! will merge once the build passes

**Review by bobfuru — APPROVED** [review]

**Comment by yporagpa** [comment]
LGTM

**Pull request merged**